src/v0/util/index.js

```diff
@@ -2258,6 +2258,14 @@ const applyCustomMappings = (message, mappings) => {
   );
 };
 
+const getRelativePathFromURL = (inputUrl) => {
```

Comment:
can you add a comment that this should only be used when URL path doesn't have any dynamic params like account ID etc

Reply:
sure

Reply:
Done